### ISSUE TITLE
Sweater Rubbing alert seems to skip first threshold "a few" more

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#318]
I noticed In version 1.4.0-dev.4 that I never seem to get "a few more negative charges" when rubbing on sweater.

There are 3 alerts (1-1, 1-3, and 1-5) described in the design document.

My intent was that alert 1-5 would trump alert 1-3 when there is a threshold change. This does not seem to occur for the first threshold, "a few". Seems to work fine for "several" and "many".

Rubbing alerts are not yet fully implemented, so this may still be in progress.

Am I supposed to hear "a few" when rubbing on sweater in this version?

**Comment by jessegreenberg** [comment]
Closing, I think this is a duplicate of #324 which was opened after this issue, and the rubbing alerts were fixed in that one.

**Comment by terracoda** [comment]
@jessegreenberg, there have been great improvements to the Sweater rubbing alerts, but this issue has not been resolved on master. The alerts still need some improvement.

I can start a new issue if needed, but thought best to re-open this one.

Note that Alert 1-1 First pick-up is working perfectly.

Here's what I am trying to get at with alerts 1-1, 1-3, and 1-5:
**Hearing:**
Alert 1-1 First pick-up is great. 

Then I hear Alert 1-3 "more charges picked up" too many times. It keeps firing until the threshold changes to "several", thus skipping "a few" all together.

Then I hear Alert 1-5 "full update" until sweater is depleted. Alert 1-5 fires when threshold changes to "several more", and then Alert 1-5 keeps firing until Sweater is depleted.

Ideally, for each new visit to sweater I should hear:

Alert 1-1, if not the last charge picked up. Then
Alert 1-3, once, if not a threshold change and if not last charge pick up
Alert 1-5, once, if not last charge. Then
Alert 1-3, until threshold change or last charge is reached. Ideally, this would not repeat more than 3 times, but I don't think we need to restrict it as long as the Alert 1-5 is heard for each threshold at least once.
Alert 1-5, on each threshold change.

Of course, the finding more charges also get mixed in which also reduces repetition of 1-3.

@jessegreenberg. The idea is to have a little back and forth between alert 1-3 and 1-5 because alert 1-5 is quite long. And we also need to get that first threshold "a few more" spoken at least once.

**Comment by jessegreenberg** [comment]
Thanks for reopening @terracoda. It seems like the main bug is that we never hear "a few" charge range, it repeats 1-3 until we hit "several" charge range. I think fixing that bug will produce the desired behavior in https://github.com/phetsims/balloons-and-static-electricity/issues/318#issuecomment-345283821.

**Comment by jessegreenberg** [comment]
Above commit makes sure that the "a few" range can be described. The problem was that we entered a new threshold range at the same time we picked up charges on entering the sweawter, so we were prioritizing alert 1-1 over alert 1-5.  Here is a walk through of the behavior now:
- Balloon is grabbed
- Balloon is off sweater
- Grabbed balloon moves onto sweater, picking up 3 charges *AND* enters the "a few" threshold range.
  - Say alert 1-1
- Balloon picks up 3 more charges in next movement
  - Say alert 1-5, because we didn't describe the threshold change on first pickup
- Now we will  say alert 1-3 until we enter a new  threshold
- When we enter a new threshold say alert 1-5

@terracoda does this seem OK?

**Comment by terracoda** [comment]
Yes, missing "a few" is the main bug. That seems perfect! I'll have a listen as soon as I can.

The second part of the bug was that I only got alert 1-5, after the "several" threshold was met.

**Comment by terracoda** [comment]
@jessegreenberg, I think your fix has fixed both issues. I still have to do a proper test with VO, but I think the alerts are correct now. I can close the issue after testing tomorrow with my regular computer.

**Comment by terracoda** [comment]
@jessegreenberg, awesome work. The sweater alerts are fixed. I just tested in Mac OS 10.13.1 High Sierra with Safari 11.0.1

Closing this issue.
